users/templates/listado.html

```diff
@@ -0,0 +1,150 @@
+                        {% else %}
+                            Inactivo
+                        {% endif %}
+                    </td>
+                    <td>
+                        {{ usuario.date_joined }}
+                    </td>
+                    <td>
+                        <div class="btn-group btn-group-sm">
+                            <div class="dropdown">
+                                <button class="btn btn-sm btn-primary btn-active-primary dropdown-toggle"
+                                        data-toggle="dropdown" type="button">
+                                    Opciones <i class="ti-angle-down"></i>
+                                </button>
+                                <ul class="dropdown-menu dropdown-menu-right">
+
+                                    {% if user.cargo == 'Administrador' %}
+
+                                        <li><a href="{% url 'usuario_perfil_usuario' usuario.pk %}">
+                                            Visualizar
+                                        </a>
+                                        </li>
+                                        <li><a href="{% url 'usuario_modificar_usuario' usuario.pk %}">
+                                            Modificar
+                                        </a>
+                                        </li>
+                                        <li><a data-toggle="modal" data-target="#modal-modificar" href="#"
+                                               data-url="{% url 'usuario_estado_usuario' usuario.pk %}"
+                                               data-listado-nombre="{{ usuario.first_name }}"
+                                               data-listado-apellido="{{ usuario.last_name }}"
+                                               data-listado-estado="{{ usuario.is_active }}"
+                                               class="modificar">
+
+                                            {% if usuario.is_active != True %}
+                                                Habilitar
+                                            {% else %}
+                                                Inhabilitar
+                                            {% endif %}
+
+                                        </a>
+                                        </li>
+                                        <li><a href="{% url 'usuarios_restablecer_contrasena' usuario.pk %}">
+                                            Reestablecer contraseña
+                                        </a>
+                                        </li>
+
+                                    {% elif user.cargo == 'Gerente' %}
+
+                                        <li><a href="{% url 'usuario_perfil_usuario' usuario.pk %}">
+                                            Visualizar
+                                        </a>
+                                        </li>
+
+                                    {% endif %}
```

Comment:
English please!

cargo is position (en otras ramas)
Usuario is user
lista_usuario

Reply:
Copy paste de lo de Ivan, no tenía ni idea que habíamos quedado en que el código en inglés XD por eso hoy elaboraremos el código de conducta para que nos "entendamos"